src/cli/runtime/http.rs

```diff
@@ -182,8 +183,41 @@ impl HttpIO for NativeHttp {
         )
         .await?)
     }
+
+    fn cl(&self) -> Box<dyn HttpIO> {
+        Box::new(self.clone())
+    }
+}
+
+pub struct ConcurrentHttp {
+    http: Box<dyn HttpIO>,
+    map: DashMap<std::thread::ThreadId, Box<dyn HttpIO>>,
```

Comment:
instead of map we can use a thread local.

Reply:
Ok